build-system/compile/bundles.config.js

```diff
@@ -920,6 +920,12 @@ exports.extensionBundles = [
     options: {hasCss: true},
     type: TYPES.MISC,
   },
+  {
+    name: 'amp-story-player',
+    version: '0.1',
+    latestVersion: '0.1',
```

Comment:
I omitted the `hasCss | cssBinaries` because we already build a CSS  (from the vanilla JS version of the amp-story-player, in `tasks/css.js`) and we would like to reuse the same CSS file. Does this seem right @ampproject/wg-infra ?

https://github.com/ampproject/amphtml/blob/5af385036ba3501358cb21adb0d8275e45afe061/build-system/tasks/css.js#L69-L73

Reply:
That's maybe a bit hacky. Can we have a `amp-story-player/0.1/amp-story-player.css` file that's just an `@import 'src/amp-story-player/amp-story-player.css'`?

Reply:
Done.